privval/file.go

```diff
@@ -429,8 +432,8 @@ func checkProposalsOnlyDifferByTimestamp(lastSignBytes, newSignBytes []byte) (ti
 	now := tmtime.Now()
 	lastProposal.Timestamp = now
 	newProposal.Timestamp = now
-	lastProposalBytes, _ := cdc.MarshalBinaryLengthPrefixed(lastProposal)
-	newProposalBytes, _ := cdc.MarshalBinaryLengthPrefixed(newProposal)
+	lastProposalBytes, _ := proto.Marshal(&lastProposal)
+	newProposalBytes, _ := proto.Marshal(&newProposal)
```

Comment:
It is unclear in many places why we were doing length prefixing and in other places not. It wasn't documented. This happened in the sdk as well.

Ill read into it to see if anything comes up

Reply:
Most likely just an oversight - it should only matter if these items are placed by themselves on the wire or in a file, or if someone needs to do random access on the object. In this specific case it certainly isn't necessary since we're just calling `bytes.Equal()` on them.